apiary.apib

```diff
@@ -3566,6 +3566,18 @@ Get a list of work orders.
 + Request (application/json)
 
     + Attributes (object)
+        + filter (object, optional)
+            + customer (object, optional)
```

Comment:
B: don't include the filter on customer yet.

Reply:
🤦‍♂